src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -457,13 +508,14 @@ final case class ProcessApiServiceImpl(
 
     val result: Future[EService] = for {
       organizationId <- getOrganizationIdFutureUUID(contexts)
-      eService       <- catalogManagementService.getEService(eServiceId)
+      eServiceUuid   <- eServiceId.toFutureUUID
+      descriptorUuid <- descriptorId.toFutureUUID
+      catalogItem    <- ReadModelQueries
+        .getEService(eServiceUuid)(readModel)
+      eService       <- catalogItem.toFuture(EServiceNotFound(eServiceId))
```

Comment:
these two expressions are used very often, we could add a little helper method for this